tests/test_rbuser_tostring.py

```diff
@@ -0,0 +1,80 @@
+                           'newbie': True,
+                           'course': 'CASE',
+                           'created': '2009-10-06 11:20:13',
+                           'altmail': 'billpbo9@mailx.dcu.net',
+                           'updated': '2015-09-28 01:42:21',
+                           'birthday': '12 02 2017',
+                           'usertype': 'member',
+                           'updatedby': 'greenday',
+                           'uidNumber': 102007,
+                           'createdby': 'greenday',
+                           'gidNumber': 1017,
+                           'yearsPaid': 1,
+                           'loginShell': '/usr/local/shells/zsh/',
+                           'objectClass': ['redbrick', 'posixAccount', 'top', 'shadowAccount'],
+                           'oldusertype': 'newb',
+                           'userPassword': 'e1NTSEF9WlBURGtJdmZJSld0WUlrak9zdGxscXYySTRRVEhBYj=',
+                           'homeDirectory': '/home/redbrick/billyx',
+                           'disuser_period': 0,
+                           'shadowLastChange': 17073, }
+        self.billy_expected_output = \
+            'cn                :  Billy Bo\n' + \
+            'id                :  15358462\n' + \
+            'uid               :  billyx\n' + \
+            'bday              :  12\n' + \
+            'host              :  carbon\n' + \
+            'year              :  3\n' + \
+            'byear             :  2017\n' + \
+            'gecos             :  new member\n' + \
+            'bmonth            :  2\n' + \
+            'course            :  CASE\n' + \
+            'newbie            :  True\n' + \
+            'passwd            :  e1NTSEF9WlBURGtJdmZJSld0WUlrak9zdGxscXYySTRRVEhBYj=\n' + \
+            'altmail           :  billpbo9@mailx.dcu.net\n' + \
+            'created           :  2009-10-06 11:20:13\n' + \
+            'updated           :  2015-09-28 01:42:21\n' + \
+            'birthday          :  12 02 2017\n' + \
+            'usertype          :  member\n' + \
+            'createdby         :  greenday\n' + \
+            'gidNumber         :  1017\n' + \
+            'uidNumber         :  102007\n' + \
+            'updatedby         :  greenday\n' + \
+            'yearsPaid         :  1\n' + \
+            'loginShell        :  /usr/local/shells/zsh/\n' + \
+            'objectClass       :  [\'redbrick\', \'posixAccount\', \'top\', ' + \
+            '\'shadowAccount\']\n' + \
+            'oldusertype       :  newb\n' + \
+            'userPassword      :  e1NTSEF9WlBURGtJdmZJSld0WUlrak9zdGxscXYySTRRVEhBYj=\n' + \
+            'homeDirectory     :  /home/redbrick/billyx\n' + \
+            'disuser_period    :  0\n' + \
+            'shadowLastChange  :  17073\n'
+
+    def test_to_string(self):
+        """Test case A. note that all test method names must begin with 'test.'"""
+        billy = rbuser.RBUser()
```

Comment:
It is a user called Billy? This literally is a representation of a Billy. I think Billy is a perfect var name.